res/css/views/right_panel/_UserInfo.scss

```diff
@@ -104,18 +104,6 @@ limitations under the License.
         height: 100% !important;
     }
 
-    .mx_UserInfo_avatar .mx_BaseAvatar_initial {
-        z-index: 1;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-
-        // override the calculated sizes so that the letter isn't HUGE
-        font-size: 6rem !important;
-        width: 100% !important;
-        transition: font-size 0.5s;
```

Comment:
Can this transition be removed because it will be implicitly done by 

```css
.mx_UserInfo_avatar > div {
    max-width: 30vh;
    margin: 0 auto;
    transition: 0.5s;
}
```

?

Reply:
(No)